config

```diff
@@ -0,0 +1,3 @@
+ {"connection": {"type": "sqlite",
```

Comment:
I thought we were getting rid of "config" default files?  This should probably match the format of the included yaml file, or at least be formatted in a way that makes sense to read.

Reply:
formatted